tree/30_generic_methods/variable_string_from_command.cf

```
@@ -53,12 +53,12 @@ bundle agent variable_string_from_command(variable_prefix, variable_name, comman
       # for example when checking promises on the server.
     pass1::
       # the cariage return is necessary if command ends by a comment
-      "temp" string => execresult("OUTPUT=${const.dollar}(/bin/true; ${command}
+      "temp" string => execresult("OUTPUT=${const.dollar}(/bin/true; 2>/dev/null ${command}
```

why are you hiding errors ?